cub/test/catch2_test_device_memcpy_batched_common.cuh

```diff
@@ -29,6 +29,19 @@ struct offset_to_ptr_op
   }
 };
 
+template <typename IteratorT, typename ValueT>
+struct skip_first_n_op
```

Comment:
Suggestion: rename to `prepend_n_constants_op`, since the operation does not skip values, but just adds more at the beginning of the sequence.